backends/beats/winlogbeat/render.go

```diff
@@ -132,6 +136,11 @@ func (wlbc *WinLogBeatConfig) RenderOnChange(response graylog.ResponseCollectorC
 		}
 	}
 
+	// gl2_source_collector can't be set with Winlogbeat so we use the shipper name
```

Comment:
we can have `winlogbeat_gl2_source_collector` but not `gl2_source_collector`. I voted for `name` then :)